app/src/sharedTest/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityTest.kt

```diff
@@ -215,8 +213,6 @@ class QuestionPlayerActivityTest {
     }
   }
 
-  // TODO(#2057): Remove when TextViews are properly measured in Robolectric.
-  @RunOn(TestPlatform.ESPRESSO) // Incorrectly passes on Robolectric and shouldn't be re-enabled
```

Comment:
Why is this TODO removed? #2057 is still open.

Reply:
Readding the todos